LIC_FILES_CHKSUM.sha256

```diff
@@ -28,3 +28,6 @@ ffa15bdce332058a03a1d923910864fb6e58bf6df66a0e3914284725b327183e  vendor/github.
 #
 # OpenLDAP Public License
 310fe25c858a9515fc8c8d7d1f24a67c9496f84a91e0a0e41ea9975b1371e569  vendor/github.com/bmatsuo/lmdb-go/LICENSE.mdb.md
+#
+# Unknown license
+2d36597f7117c38b006835ae7f537487207d8ec407aa9d9980794b2030cbc067  vendor/golang.org/x/text/LICENSE
```

Comment:
I couldn't figure out the actual license. Any suggestion?

Reply:
This is BSD-3-Clause.